### ISSUE TITLE
Running Newpipe in Background While Closing out the App. 

### ISSUE THREAD

**Issue by RSicario** [TeamNewPipe/NewPipe#11089]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

I would like for Newpipe content to keep playing in the background after I have closed the app in the app overview (the 3 bar "button"/icon on the bottom left). This feature worked well without a problem on my Samsung A12 but doesn't appear to work on my new Samsung A13. 

There is an app called Music Player Go that I downloaded from F-Droid that allows me to do this. I can close the app while music is playing and the music does not stop. When I access the pull down menu by swiping from the top of the phone I have a simple preview bar with pause play for Music Player Go. I used to be able to do the same with Newpipe on my Samsung A12. I can no longer do this with the A13 for Newpipe. 

I have attached screenshots to help better visualize what I mean. I used Music Player Go because it functions exactly the way that I am requesting Newpipe work again. 

### Why do you want this feature?

There would be less UI clutter. There would be less apps to swipe through when I hit the android app overview "button" (the three bars on the bottom left of my Samsung A13). This allows me to scroll through only the high priority apps I want access to. 

### Additional information

![photo_1_2024-05-22_11-17-19](https://github.com/TeamNewPipe/NewPipe/assets/170545146/4c39a0c0-44aa-4ec7-9d97-3395832dd35a)
![photo_2_2024-05-22_11-17-19](https://github.com/TeamNewPipe/NewPipe/assets/170545146/5f906a60-bec0-45f1-ad0d-19a2f184c89e)
![photo_3_2024-05-22_11-17-19](https://github.com/TeamNewPipe/NewPipe/assets/170545146/52e98695-360d-47ab-bc68-e16bd0c1235f)
![photo_4_2024-05-22_11-17-19](https://github.com/TeamNewPipe/NewPipe/assets/170545146/ad0d7a1e-6e51-49a0-b0a4-2774bacab249)
![photo_5_2024-05-22_11-17-19](https://github.com/TeamNewPipe/NewPipe/assets/170545146/bdcb45c6-d3ac-4131-84e4-273934516157)

**Comment by RSicario** [comment]
As I described, I can play it in the background as long as the app is open
in the background too. However if I close the app from the app overview
(the three bars in the lower left) the app closes completely. Newpipe stops
playing in the background altogether. It didnt do that on my Samsung A12.
It's doing it on the Samsung A13.  Music PLayer Go, an app I got off of
F-Droid, managed to preserve this functionality between the A12 and A13. I
would like to be able to close the app in app overview without having to
lose background.

**Comment by RSicario** [comment]
Yes that works. I never had to use the background button in the a12. it was
automatic. i dont want to have to go back into the app to do that when i
want to close all apps at the same time. its not laziness, like i said it's
about streamlining the apps in the app preview window. i usually need that
while im driving. if im at a red light, it makes scrolling through a maps
app and the phone app quicker.

**Comment by RSicario** [comment]
It worked better before. didnt have to hit background for it to keep
playing. it was more natural. if there could be some kind of reversion that
would be great.

**Comment by RSicario** [comment]
Its not laziness... imagine driving and you forgot to press the background
button and you have 15 apps open. The most efficient way in during a
redlight stop is to hit close all apps... what if i have something playing?
at a red light i really wouldnt like to be stressed. what if i accidentally
close out all apps?  it's just *smoother*. it's passively safer too and
intuitive. not being on auto is makes the app mechanical instead of useful
and helpful.

**Comment by RSicario** [comment]
thats not it. put myself in a scenario here.  ur in a rush and have to pull
out of ur drive way (to get to some appointment) or are at a redlight, and
you want to just keep the audio playing in the background. your goal is to
spend as little time as possible on the phone. u want to closeout all the
apps quickly without having to let go of the background content playing.
maybe you just want only the GPS app or just the newpipe in background with
the contact list open (to call your kid's school). or if a student just got
out of his part time job and has to rush over to class and listen to a
lecture on newpipe and is running low on battery power, why should he or
she closing out all their apps have to eliminate the newpipe background
feature? how is any of that laziness?* it minimizes screen time.* how does
reducing screen time make driving less safe? the idea is to not interact
with your phone at all, and if for some reason you have to, and as human
beings we all break these rules at certain times, and not necessarily
willingly, wouldnt you want your experience to be smoother and minimal with
the app while driving?

sometimes i genuinely forget to press the background button but i'll do it
at a red light (its reasonably enough time to take care of small stuff like
closing out apps). i think it *enhances *safety rather than compromising
it. it turns into play, close and forget. think about it, it's very
facilitative. hell, if you dont believe me, change the code for yourself
exclusively and try it out for a month or two. I guarantee you're gonna
think its a smoother and superior experience. it seems trivial but
seamlessness is unrealized and unappreciated perfection until its gone. its
not trivial. give it a shot. code the change for yourself and see how ur
experience differs. you'll like newpipe more for how perfectly it operates.

**Comment by ChiefMikeK** [comment]
A couple of options :neckbeard: 
- Use the play in background, switch to your contacts app, then turn off the screen.
- Only have 2 or 3 apps open and use the samsung switch/swap app function.
- Download the audio file(s) and use another app to play.
- 
![Screenshot_samsungButtons](https://github.com/TeamNewPipe/NewPipe/assets/20070053/5d50e45b-f2f7-4cf3-8ea5-fbbf95534668)

**Comment by RSicario** [comment]
Alright. Its not just that. Its also forgefulness. sometimes people forget
to follow procedure when closing all the apps out. Anyway, like i said it
would make using the app smoother.

thanks.

**Comment by uestueab** [comment]
Is this still not implemented? The code to do so is literally there right? 

If i have this setting below, why would i still need to hit background play to achieve what he wishes for in his feature request? The current flow feels inconsistent and leaves room for confusion. Leaving it as is would be too high a level of laziness to account for.

<img width="1079" height="1189" alt="Image" src="https://github.com/user-attachments/assets/80da02f6-09f4-46f1-8fc0-ee547f5a8ce8" />
